packages/did/src/DidResolver/Resolver.spec.ts

```diff
@@ -271,16 +276,18 @@ describe('When resolving a service endpoint', () => {
   })
 
   it('throws for invalid URIs', async () => {
-    const uriWithoutFragment = getKiltDidFromIdentifier(
+    const uriWithoutFragment: DidUri = getKiltDidFromIdentifier(
       deletedIdentifier,
       'full'
     )
     await expect(
+      // @ts-ignore
```

Comment:
This `@ts-ignore` is also warned by my compiler. We should probably ignore it high up in the file.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/505/commits/29da2de6f66b7e9f7251ca6af9f1d3a6fedc07b9